apps/heureka/src/components/Services/ServicesList/ServicesList.tsx

```diff
@@ -21,6 +23,34 @@ export const ServicesList = ({ filterSettings }: ServiceListProps) => {
     filterSettings,
   })
 
+  const [selectedService, setSelectedService] = useState<ServiceOverViewPanelType | null>(null)
```

Comment:
we need to move `useFetchServices` to `Services` otherwise the list page will keep rerendering on this state change.